apps/examples/testcase/le_tc/kernel/tc_environ.c

```diff
@@ -150,21 +150,21 @@ static void tc_environ_putenv(void)
 	TC_ASSERT_EQ("putenv", ret_chk, OK);
 
 	psz_getvalue = getenv("PATH");
-	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, clearenv());
-	TC_ASSERT_EQ_CLEANUP("getenv", strcmp(psz_getvalue, "C:"), 0, clearenv());
+	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, unsetenv("PATH"));
+	TC_ASSERT_EQ_CLEANUP("getenv", strcmp(psz_getvalue, "C:"), 0, unsetenv("PATH"));
 
 	/* Changing the value of already existing PATH variable */
 	ret_chk = putenv("PATH=D:");
 	TC_ASSERT_EQ("putenv", ret_chk, OK);
 
 	psz_getvalue = getenv("PATH");
-	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, clearenv());	
-	TC_ASSERT_EQ_CLEANUP("getenv", strcmp(psz_getvalue, "D:"), 0, clearenv());
+	TC_ASSERT_NEQ_CLEANUP("getenv", psz_getvalue, NULL, unsetenv("PATH"));	
```

Comment:
I wonder, Does it still override the  default value of PATH environment ? Do we need to take backup and restore it before changing the PATH environment ?

Reply:
if not restore, it will take some memory space. So I think we should restore